lib/media_types/dsl.rb

```diff
@@ -37,6 +37,8 @@ def valid_unsafe?(output, media_type = to_constructable, **opts)
       end
       
       def validate!(output, **opts)
+        target_scheme = self.media_type_validations.scheme
+        assert_sane! unless target_scheme.asserted_sane
```

Comment:
Methods that answers a question should end with `?`

```suggestion
        assert_sane! unless target_scheme.asserted_sane?
```